packages/core/src/components/graphs/scatter.ts

```diff
@@ -58,6 +58,7 @@ export class Scatter extends Component {
 			.classed("filled", filled)
 			.classed("unfilled", !filled)
 			.attr("cx", (d, i) => this.services.cartesianScales.getDomainValue(d, i))
+			.attr("cy", 311)
```

Comment:
```suggestion
			.attr("cy", this.services.cartesianScales.getRangeScale().range()[0])
```
If we use the scale funtions we dont have to hardcode the 311 (magic number?) .. 
we can use the bound on the range scale to get where the bottom x axis is (regardless of whether we are using a top/bottom axes). Just depends if we want bottom up transtiton even if the user has a top labelled axis chart